### ISSUE TITLE
[Task] Cluster README

### ISSUE THREAD

**Issue by Rui-Pedro-Pires** [SEAME-pt/Team02-Cluster#24]
### Description

Create the README for the Cluster Repo.

### Priority

High

**Comment by ziliolu** [comment]
Readme branch was correctly merged into dev branch. THere are a few things that are still missing in the readme file like gifs and Usage section that will be approached later once the project is finished.
